jhu-idc/idc-isle-dc
# Requested config changes from Katie

This does three things: 
 
 * adds `Held by` as an option for Relationship Type on the `Contributor` field (of a Repository Item)
 * adds new field called `Custodial History` to the Repository Item Content Type
 * adds back in a missing URL in a Islandora Media Use tax term (FITS File)

To test
 * add a Repository Item, ensure that the option of `Held by` shows up as a choice in the `Relationship Type` drop down.  Choose it and create an item from it. 
 * Add information in the Custodial History text box as well. (or just run tests - which covers this field).
 * Test out json-ld formatting by adding `?_format=jsonld` to the node's url and ensure that you see a triple for `heldBy`

```
"http://id.loc.gov/ontologies/bibframe/heldBy": [

    {
        "@id": "http://islandora-idc.traefik.me/taxonomy/term/40?_format=jsonld"
    }
],

And one for Custodial History: 
```
"http://id.loc.gov/ontologies/bibframe/custodialHistory": [

    {
        "@value": "This is custodial history",
        "@language": "eng"
    }
],

To test the Media Use fix, see that there is a URL in the `FITS File` Islandora Media Use taxonomy term. Also, add an Image to a Repository Item.  Tag the media as `Original File`.  Save it and wait. Ensure that a FITS derivative shows up in a few seconds.  If it does, than this is fixed. 

**Please note: I did not adjust the migrations for media at all -- when one does a `make config-export` drupal changes the formatting of those files.**  

Resolves jhu-idc/iDC-general#339
Resolves #112
